minecraft_servers/suchtcraft/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "server_name": "suchtcraft",
+  "nice_name": "SuchtCraft",
+  "direct_ip": "SuchtCraft.net",
+  "social": {
+    "web": "https://suchtcraft.net/",
+    "web_support": "https://suchtcraft.net/forum/",
+    "discord": "https://suchtcraft.net/discord",
```

Comment:
"Own urls" are only supported for partners. Please use the official discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url

Reply:
Changed with 1f74679de7ddc16a9434201b6b636a6c464ce71a